### PR TITLE
add chrome driver versions and notice for w3c

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

When I investigated https://github.com/appium/python-client/issues/234, I noticed chrome driver hasn't supported W3C fully yet.
Thus, in this PR, I'd like to mention the W3C related known issue in `chromedriver.md`.

I also append some chrome driver/chrome versions following existing documentation.
refer: https://chromium.googlesource.com/chromium/src/+refs

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/web/chromedriver.md [hunk 1]
```diff
@@ -137,3 +141,8 @@ CHROMEDRIVER_CDNURL=http://npm.taobao.org/mirrors/chromedriver npm install appiu
 
 It may also be necessary to adjust network proxy and firewall settings to allow
 the download to occur.
+
+### W3C support
+Chrome Driver hasn't followed the W3C standard yet. If you encounter proxy command error like [this issue](https://github.com/appium/python-client/issues/234), please update your chrome driver version.
```
Comment (imurchie) [inline docs/en/writing-running-appium/web/chromedriver.md]: Chromedriver is a singe word, capitalized at the beginning. Might as well stay consistent.

**Review by imurchie — APPROVED** [review]
Thanks for this!

**Comment by KazuCocoa** [comment]
thanks!

**Review by jlipps — COMMENTED** [review]
thanks for the explanation @KazuCocoa

**Pull request merged**